docs/flowRetry.rst

```diff
@@ -0,0 +1,54 @@
+.. _flow-level-retry:
+
+Flow Level Retry
+================
+
+**Authors**:
+`Zhi Zeng <https://github.com/tsangz2013>`_
+
+Ref: :ref:`ContainerizationDesignDoc`
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 1
+
+Background
+----------
+As Azkaban moves into containerization, executions are observing some k8s pod related / infra
+related issues that result in the whole execution falls. Plus, many of the flows takes hours to
+days to run, and if not attended, a failure may takes hours before the user restart it.
+
+So we build the feature that allows users to enable auto-retry on flow-level,
+as well as opt-in-able strategies how to restart the execution.
```

Comment:
s/"as well as opt-in-able strategies how to restart the execution"/"as well as opt-in-able strategies on how to restart the execution"